net-misc/anydesk/anydesk-5.4.1.ebuild

```diff
@@ -0,0 +1,106 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+inherit desktop systemd xdg-utils
+
+DESCRIPTION="Feature rich multi-platform remote desktop application"
+HOMEPAGE="https://anydesk.com"
+SRC_URI="amd64? ( https://download.anydesk.com/linux/${P}-amd64.tar.gz )
+	x86? ( https://download.anydesk.com/linux/${P}-i386.tar.gz )"
+
+# OpeSSL/SSLeay, libvpx, zlib, Xiph, xxHash
+LICENSE="AnyDesk-TOS BSD BSD-2 openssl ZLIB"
+SLOT="0"
+KEYWORDS="-* ~amd64 ~x86"
+
+RDEPEND="
+	dev-libs/atk
+	dev-libs/glib:2
+	media-libs/fontconfig:1.0
+	media-libs/freetype:2
+	media-libs/glu
+	media-libs/mesa[X(+)]
+	sys-auth/polkit
+	x11-libs/cairo
+	x11-libs/gdk-pixbuf:2
+	x11-libs/gtk+:2
+	x11-libs/gtkglext
+	x11-libs/libICE
+	x11-libs/libSM
+	x11-libs/libX11
+	x11-libs/libxcb
+	x11-libs/libXdamage
+	x11-libs/libXext
+	x11-libs/libXfixes
+	x11-libs/libXi
+	x11-libs/libxkbfile
+	x11-libs/libXmu
+	x11-libs/libXrandr
+	x11-libs/libXrender
+	x11-libs/libXt
+	x11-libs/libXtst
+	x11-libs/pango
+	x11-libs/pangox-compat
+"
+
+RESTRICT="bindist mirror"
+
+QA_PREBUILT="opt/${PN}/*"
+
+src_install() {
+	local dst="/opt/${PN}"
```

Comment:
I find this a bit redundant, but I dont mind about it too much.

Rest looks good!